README.md

```diff
@@ -26,15 +30,16 @@ Your directory structure should look something like this:
 | - SimpleBook
 ```
 
-4. (Optional) Populate necessary environment variables
+4. (Optional, but highly recommended) Populate necessary environment variables
 
 ```
-$ cp .env.template .env
-$ vi .env
-$ source .env
+$ cp configuration/configuration.env.template configuration/configuration.env
+$ vi configuration/configuration.env
```

Comment:
A good thing to use is `$EDITOR` here, as it's common in *nix to bind that variable to whichever editor is your preference, which will make copy pasting easier.  I know you were just updating the commands that were there before, but it would be a nice improvement :)

Reply:
Happy to make that change. I was just copying from the previous work by using `vi`. Of course, I'm a `vi` user, so I had no problem with it. :-) I will make the change.